decisions/13-schemaorg-identifier-as-PropertyValue.md

```diff
@@ -1,5 +1,7 @@
 # schema.org/identifier as schema.org/PropertyValue over Text/URL #
```

Comment:
See comments on issue #13 - further specification here is needed.

Reply:
Agreed we need to discuss that, but let's do so over in issue #13.  All I did in this PR was to rename the ADR file to match the convention proposed in the new workflow.  So I'm going to resolve this comment.